web/pageObjects/register_with_email_page.ts

```diff
@@ -0,0 +1,99 @@
+import * as defaultPage from "./default_page"
+
+const pageHeader = '//main//h1'
+const privacyPolicyHeader = '//main//h1[1]'
+const enterEmailField = '//main//input[@name="email"]'
+const enterPassField = '//main//input[@name="password"]'
+const enterPassRepeatField = '//main//input[@name="passwordRepeat"]'
+const btnSignUp = '//main//button[@type="submit"]'
+const emailAlreadyExistsMsg = '//main//form//h3'
+const emailValidationMsg = '//main//form//div//div[1]//h3'
+const passValidationMsg = '//main//form//div[2]/h3'
+const passRepeatValidationMsg = '//main//form//div[3]/h3'
+const linkSignIn = '//main//a[@href="/sign-in"]'
+const linkPrivacyPolicy = '//main//a[text()="communications"]'
+
+//----------Actions----------//
+
+export async function signUpWithEmail(email: string, pass: string, repeatPass: string) {
+    await defaultPage.enterTextByLocator(enterEmailField, email)
+    await defaultPage.enterTextByLocator(enterPassField, pass)
+    await defaultPage.enterTextByLocator(enterPassRepeatField, repeatPass)
+    await defaultPage.clickByLocator(btnSignUp)
```

Comment:
Can reuse function here and in other steps clickSignUpBtn() which you already have :))